docs/content/blog/authelia-traefik-setup-guide/index.md

```diff
@@ -166,30 +192,11 @@ tls:
         - TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305
         - TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305
 ```
-Create acme.json with 600 permissions before starting Traefik: `touch acme.json && chmod 600 acme.json`
+Note: Create acme.json with 600 permissions before starting Traefik.
```

Comment:
Is this explicitly necessary? I believe Traefik creates this upon creation of it's certificate store.

Reply:
I have had issues where Traefik generates acme.json as a directory rather than a json file.